### DISCUSSION TITLE
i cant fix it

### DISCUSSION THREAD

**Discussion by MrMajonez** [FabricMC/fabric-api#2961] [Player support]
i cant fix it
![image](https://user-images.githubusercontent.com/127629223/224510722-04e38470-ba39-4957-be57-099f52df6af5.png)

**Comment by Daomephsta** [comment]
The errors say Phosphor is incompatible with any version of Starlight, and Starlight is incompatible with any version of Phosphor.  
So you cannot have both installed, you must pick one and install only that one.
